src/blinker/_utilities.py

```diff
@@ -93,3 +98,35 @@ def __get__(self, obj, cls):
         value = self._deferred(obj)
         setattr(obj, self._deferred.__name__, value)
         return value
+
+
+def is_coroutine_function(func: Any) -> bool:
```

Comment:
Please add TODO comment, scheduling this to be removed with Python 3.7 support

Reply:
I'm not sure it is a TODO, as nothing can be done at the moment. (I tend to see TODOs as indicating something that would have been done if time allowed). This said, I don't mind so if you still think a TODO should be added please say so.

Reply:
ok, it's no big deal, let's leave it as it is